pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: pdftron_flutter
 description: A convenience wrapper to build Flutter apps that use the PDFTron mobile SDK for smooth, flexible, and stand-alone document viewing.
-version: 1.0.0-beta.57
+version: 1.0.0-beta.58
```

Comment:
don't think we need to change this because we likely don't need to publish a new version on pub.dev with no changes